tsconfig.refs.json

```diff
@@ -0,0 +1,125 @@
+{
```

Comment:
It looks like this file may have been accidentally included in this PR?

Reply:
Huh, not sure where that came from, or why the .GITIGNORE didn't catch it. I have removed it though.